iocore/net/P_SNIActionPerformer.h

```diff
@@ -168,16 +236,19 @@ class TunnelDestination : public ActionItem
             to = (port - pos) - 1;
           }
         }
-        const auto &number_str = dst.substr(pos + 1, to);
+        std::string_view number_str{dst.substr(pos + 1, to)};
         if (!is_number(number_str)) {
           // it may be some issue on the configured string, place the char and keep going.
           real_dst += *c;
           continue;
         }
-        const std::size_t group_index = std::stoi(number_str);
+        const std::size_t group_index = swoc::svtoi(std::string{number_str});
```

Comment:
The point of using `svtoi` is to avoid converting to `std::string`. E.g. `swoc::svtoi(number_str)`

Reply:
Right, that's nice.